util/time_test.go

```diff
@@ -0,0 +1,87 @@
+package util
+
+import (
+	"fmt"
+	"testing"
+)
+
+//要考虑时区
```

Comment:
It's better to write English comments here :)

Reply:
ok

Reply:
already fix